website/specs/cdsc-wg1-02/overview.md

```diff
@@ -8,10 +8,515 @@ meta_description: Linux Foundation Energy - Carbon Data Specification Consortium
+Content-Type: application/json;charset=UTF-8
+
+{
+    # OAuth Metadata
+
+    ## OAuth Metadata - General information
+    "issuer": "https://demoutility.com",
+    "service_documentation": "https://demoutility.com/docs/oauth",
+    "op_policy_uri": "https://demoutility.com/legal/oauth-policy",
+    "op_tos_uri": "https://demoutility.com/legal/oauth-terms",
+
+    ## OAuth Metadata - Functional endpoints
+    "registration_endpoint": "https://demoutility.com/oauth/register",
+    "authorization_endpoint": "https://demoutility.com/oauth/authorize",
+    "token_endpoint": "https://demoutility.com/oauth/token",
+    "revocation_endpoint": "https://demoutility.com/oauth/token/revoke",
+    "introspection_endpoint": "https://demoutility.com/oauth/token/info",
+    "pushed_authorization_request_endpoint": "https://demoutility.com/oauth/par",
+    "code_challenge_methods_supported": [
+        "S256"
+    ],
+
+    ## OAuth Metadata - Scopes available
+    "scopes_supported": [
+        "client_admin",
+        "usage",
+        ...
+    ],
+    "authorization_details_types_supported": [
+        "client_admin",
+        "usage",
+        ... (same as "scopes_supported")
+    ]
+
+
+    # CDS Extension
+    "cds_oauth_version": "v1",
+
+    # CDS Extension - Required human-method alternative links
+    "cds_human_registration": "https://demoutility.com/clients/register",
+    "cds_human_directory": "https://demoutility.com/clients/directory",
+    "cds_test_accounts": "https://demoutility.com/docs/oauth/test-accounts",
+
+    # CDS Extension - Required API endpoints
+    "cds_clients_api": "https://demoutility.com/api/clients",
+    "cds_client_settings_api": "https://demoutility.com/api/client-settings",
+    "cds_client_updates_api": "https://demoutility.com/api/client-updates",
+    "cds_scope_credentials_api": "https://demoutility.com/api/scope-credentials",
+    "cds_grants_api": "https://demoutility.com/api/grants",
+    "cds_directory_api": "https://demoutility.com/api/directory",
+
+    # CDS Extension - scope descriptions and requirements
+    "cds_scope_descriptions": {
+
+        "client_admin": {
+            ## Information about the scope
+            "id": "client_admin",
+            "name": "Client Administrator Access",
+            "description": "Allows ability to view and manage the requesting client's registration.",
+            "documentation": "https://demoutility.com/docs/oauth/scopes#client_admin",
+
+            ## Extra fields required for registration
+            "registration_requirements": [],
+
+            ## OAuth access configuration for this scope
+            "response_types_supported": [],
+            "grant_types_supported": ["client_credentials"],
+            "token_endpoint_auth_methods_supported": ["client_secret_basic"],
+            "code_challenge_methods_supported": [],
+
+            ## Rich Authorization Request 
+            "authorization_details_fields": [],
+        },
+
+        "bills": {
```

Comment:
From @antarasargam ([on April 24, 2024](https://github.com/carbon-data-specification/Customer-Data/pull/25#discussion_r1578417513)):

---

nit: But the scope used here `bills` does not match the `scopes_supported` array where the scope used is `usage` instead

---